eta/core/video.py

```diff
@@ -4139,6 +4137,16 @@ def run(self, inpath, outpath, verbose=False):
         else:
             in_opts = self._in_opts
 
+        # Automatically determine the starting number of the inpath for cases
+        # where it's a sequence pattern (e.g. %06d.jpg).  The default behavior
+        # from ffmpeg is to start at 0 and look in the range [0,4].  If the first
+        # matched pattern begins above 4 we want to explicitly set that.
+        if "-start_number" not in in_opts and is_supported_image_sequence(
+            inpath
+        ):
+            start_number = next(iter(etau.parse_pattern(inpath)), 0)
```

Comment:
Since `FFmpeg` is supposed to be a very low-level wrapper around `ffmpeg`, consider putting the `start_number` computation inside a `try-except` so that, if we fail to guess a start number, the command runs nonetheless, in case `ffmpeg` is smarter? (perhaps with a warning logged).

Reply:
Ah missed the `0`. My bad